.github/workflows/wp-scripts-lint.yml

```diff
@@ -82,6 +94,18 @@ jobs:
               .parse(process.env.ENV_VARS)
               .forEach(envVar => core.exportVariable(envVar.name, envVar.value));
 
+      - name: Set up SSH
+        if: ${{ env.GITHUB_USER_SSH_KEY != '' }}
+        uses: webfactory/ssh-agent@v0.8.0
+        with:
+          ssh-private-key: ${{ secrets.GITHUB_USER_SSH_KEY }}
+
+      - name: Setup Git
+        if: ${{ env.GITHUB_USER_EMAIL != '' && env.GITHUB_USER_NAME != '' }}
```

Comment:
`GITHUB_USER_EMAIL` and `GITHUB_USER_NAME` are required, so there is no need to check them here again.
```suggestion
```

Reply:
yes, I copied most of the code from https://github.com/inpsyde/reusable-workflows/blob/f0fbb4ebdb4628bb40f9a949c3646720875e0e6a/.github/workflows/build-assets-compilation.yml. But secret definitions from https://github.com/inpsyde/reusable-workflows/blob/f0fbb4ebdb4628bb40f9a949c3646720875e0e6a/.github/workflows/build-and-push-assets.yml#L55-L63 :smile: . I will update PR to make all new secrets optional.